compiler/frontend/src/org/jetbrains/kotlin/resolve/AnnotationChecker.kt

```diff
@@ -221,9 +221,18 @@ class AnnotationChecker(
             }.toSet()
         }
 
-        fun getDeclarationSiteActualTargetList(annotated: KtElement, descriptor: ClassDescriptor?, trace: BindingTrace):
+        fun getDeclarationSiteActualTargetList(
```

Comment:
I don't like an idea to change front-end code in this commit: as result we get an additional parameter used only from IDE code. I think it's better to call `getActualTargetList` from fix (make it public for this), and then take necessary targets from this list (effectively inlining your function here). As alternative idea, you can write separate function in IDE for a case with `all = true`.